aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -130,6 +130,10 @@
       "type": "boolean",
       "description": "A value that indicates whether minor engine upgrades are applied automatically to the DB instance during the maintenance window. By default, minor engine upgrades are applied automatically."
     },
+    "AutomaticBackupReplicationRegion": {
+      "type": "string",
+      "description": "Enables replication of automated backups to a different Amazon Web Services Region."
+    },
```

Comment:
I believe we are missing the new permissions of newly added calls: StartDBInstanceAutomatedBackupsReplication, StopDBInstanceAutomatedBackupsReplication.

Please make sure you follow the process of adding new permissions.